src/stacks-hierarchy/Root_PurchaseOverviewScreen/navigation-types.ts

```diff
@@ -10,6 +11,8 @@ export type Root_PurchaseOverviewScreenParams = {
   userProfilesWithCount?: UserProfileWithCount[];
   fromTariffZone?: TariffZoneWithMetadata;
   toTariffZone?: TariffZoneWithMetadata;
+  fromHarbor?: StopPlace;
+  toHarbor?: StopPlace;
```

Comment:
Can this be typed a little more declarative? There is no way both `fromTariffZone`/`toTariffZone` and `fromHarbor`/`toHarbor` can be specified at the same time, and I think that can be expressed in the typing.